src/lib/stdout-manipulator.ts

```diff
@@ -54,7 +54,10 @@ function color(line: string, noClear: boolean = false): string {
   return line;
 }
 
-export function print(noColors: boolean, noClear: boolean, line: string): void {
+export function print(noColors: boolean, noClear: boolean, line: string, signalEmittedFiles: boolean): void {
```

Comment:
maybe it's time to move `line: string` to be the first parameter?

Reply:
@gilamran Definitively! What do you think of `print(lint, { optionalParam: bool })` format? 81c3159fed10c89b3bbb1a2d653f5f129237fa00
I can remove the ability to make them optional if you prefer as they are explicit in the only usage.